### PR TITLE
Add recipe for kele

### PR DESCRIPTION
**Pull request by jinnovation**
### Brief summary of what the package does

Kele (kě lè, or kə-ˈlə) (“Kubernetes Enablement Layer for Emacs”) is a Kubernetes cluster management package. It enables you to perform operations as coarse or fine-grained as you need, fast, and get back to your work as quickly as possible.

For more details, see: https://jonathanj.in/kele.el.

For comparisons to other MELPA packages of this one's ilk, e.g. `kubel` (#5873) and `kubernetes-el` (#4666), see: https://jonathanj.in/kele.el/explanations/comparisons.

### Direct link to the package repository

https://github.com/jinnovation/kele.el

### Your association with the package

I am the maintainer and author.

### Relevant communications with the upstream package maintainer

**None needed**

### Checklist

- [x] The package is released under a [GPL-Compatible Free Software License](https://www.gnu.org/licenses/license-list.en.html#GPLCompatibleLicenses)
- [x] I've read [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)
- [x] I've used the latest version of [package-lint](https://github.com/purcell/package-lint) to check for packaging issues, and addressed its feedback
- [x] My elisp byte-compiles cleanly
- [x] `M-x checkdoc` is happy with my docstrings
- [x] I've built and installed the package using the instructions in [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)

### CODE REVIEW

File: recipes/kele [hunk 1]
```diff
@@ -0,0 +1 @@
+(kele :repo "jinnovation/kele.el" :fetcher github :files (:defaults))
```
Comment (riscy) [inline recipes/kele:1]: Prefer the default recipe if possible:
```suggestion
(kele :repo "jinnovation/kele.el" :fetcher github)
```

**Review by riscy — COMMENTED** [review]
Thanks for this, and for the thoughtful documentation.  Here's a quick first pass --

`kele.el` with byte-compile using Emacs 28.1:
```
kele.el:108:60: Warning: Unused lexical variable `error-thrown'
In kele-namespace-switch-for-current-context:
kele.el:276:43: Warning: reference to free variable `context'
In kele--namespace-annotate:
kele.el:278:34: Warning: Unused lexical argument `namespace-name'
```
You can pacify the compiler re: error-thrown by renaming that to `_error-thrown`, similarly with `namespace-name`.  The `context` one looks like the variable is genuinely missing in that scope.

Package and license:
- Add *formal* license boilerplate or an [SPDX-License-Identifier](https://spdx.dev/ids/) to kele.el

**Comment by jinnovation** [comment]
> You can pacify the compiler re: error-thrown by renaming that to _error-thrown, similarly with namespace-name. The context one looks like the variable is genuinely missing in that scope.

Thanks, these should be taken care of now.

> Add formal license boilerplate or an [SPDX-License-Identifier](https://spdx.dev/ids/) to kele.el

Done.

**Comment by jinnovation** [comment]
Hi @riscy, no rush at all but would love a re-review when you get a spare second. 👋 Cheers.

**Comment by riscy** [comment]
Taking a look - I see there's another file added which needs license boilerplate as well.  After that this looks like it should be a quick merge.

**Comment by jinnovation** [comment]
> Taking a look - I see there's another file added which needs license boilerplate as well. After that this looks like it should be a quick merge.

Thanks @riscy, apologies for the back-and-forth on this.

I've added a `SPDX-License-Identifier` header to `kele-fnr.el`; let me know if that's sufficient!

**Pull request merged**